dotCMS/dependencies.gradle

```diff
@@ -17,8 +17,8 @@ dependencies {
     compile group: 'com.dotcms.lib', name: 'dot.axiom-impl', version:'1.2.5_2'
     compile group: 'com.dotcms.lib', name: 'dot.bsf', version:'ukv_2'
     compile group: 'com.dotcms.lib', name: 'dot.bsh', version:'2.0b4_3'
-    compile 'net.bytebuddy:byte-buddy:1.6.12'
-    compile 'net.bytebuddy:byte-buddy-agent:1.6.12'
+    compile 'net.bytebuddy:byte-buddy:1.9.0'
```

Comment:
Wouldn't this require to change the jar at `docker` repo? Or is it just for compile purposes?